pycalphad/core/composition_set.pyx

```diff
@@ -0,0 +1,104 @@
+from pycalphad.core.phase_rec cimport PhaseRecord
+cimport numpy as np
+import numpy as np
+
+cdef public class CompositionSet(object)[type CompositionSetType, object CompositionSetObject]:
```

Comment:
Add documentation for the CompositionSet addressing its purpose. Maybe highlight use in miscibility gaps.